incubator/hnc/internal/validators/object_test.go

```diff
@@ -308,6 +309,20 @@ func TestUserChanges(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "Deny creation of object with invalid treeSelect annotation",
```

Comment:
Probably also a good idea to have an "allow" test.

Reply:
Added